src/NexusMods.App.UI/Overlays/AlphaWarning/AlphaWarningView.axaml

```diff
@@ -0,0 +1,83 @@
+<reactive:ReactiveUserControl
+    x:TypeArguments="local:IAlphaWarningViewModel"
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:reactive="http://reactiveui.net"
+    xmlns:local="clr-namespace:NexusMods.App.UI.Overlays.AlphaWarning"
+    xmlns:base="clr-namespace:NexusMods.App.UI.Overlays.Generic.MessageBox.Base"
+    xmlns:icons="clr-namespace:NexusMods.Icons;assembly=NexusMods.Icons"
+    mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="600"
+    x:Class="NexusMods.App.UI.Overlays.AlphaWarning.AlphaWarningView">
+
+    <base:MessageBoxBackground MinWidth="576" MaxWidth="576" MinHeight="523" MaxHeight="523">
+        <base:MessageBoxBackground.TopContent>
+            <StackPanel Orientation="Vertical" Margin="16, 24, 24, 24" Classes="Spacing-3_5">
+                <StackPanel Classes="Spacing-4">
+                    <TextBlock Classes="HeadingXSSemi">Thank you for joining the Nexus Mods App alpha test</TextBlock>
+
+                    <Border Classes="Rounded-lg WarningSubdued20" Padding="16">
+                        <StackPanel Spacing="8">
+                            <StackPanel Orientation="Horizontal" Spacing="2">
+                                <icons:UnifiedIcon Size="20" Classes="Alert"/>
+                                <TextBlock Classes="BodyLGBold">Important</TextBlock>
+                            </StackPanel>
+
+                            <TextBlock Classes="BodyMDNormal" TextWrapping="Wrap">This app is still very early in development and currently only supports Stardew Valley. Please ensure no mods are installed in your game folder before testing.</TextBlock>
```

Comment:
Technically speaking these should be localized, given we will likely reuse this dialog for beta; which may have non-English people trying to participate, but it's not big enough a deal to block the PR.